changelogs/unreleased/bump-compiler-version-for-release.yml

```
@@ -0,0 +1,4 @@
+---
+description: "Bump compiler version to 2025.3"
+change-type: patch
+destination-branches: [iso8]
```

```suggestion
destination-branches: [iso8, master]
```

I'd keep it simple and include master for now. Ideally, when we first release master we'll bump it to 2026, or drop it altogether, but for now let's just make sure that we don't start trailing behind.